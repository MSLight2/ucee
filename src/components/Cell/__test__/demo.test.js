import { demoTest } from '../../../tests/shared/demoTest';
import CellDemo from '../demo';

demoTest('Cell', CellDemo);
