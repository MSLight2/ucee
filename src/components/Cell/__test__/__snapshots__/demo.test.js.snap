// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell demo test 1`] = `
Array [
  <div
    className="ucee-cell-group"
  >
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
    </div>
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
        <div
          className="ucee-cell__label"
        >
          描述信息
        </div>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
    <div
      className="ucee-cell ucee-cell--center"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
        <div
          className="ucee-cell__label"
        >
          描述信息
        </div>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
  </div>,
  <div>
    large
  </div>,
  <div
    className="ucee-cell-group"
  >
    <div
      className="ucee-cell ucee-cell--large"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
    <div
      className="ucee-cell ucee-cell--large"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
        <div
          className="ucee-cell__label"
        >
          描述信息
        </div>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
  </div>,
  <div>
    right-icon
  </div>,
  <div
    className="ucee-cell-group"
  >
    <div
      className="ucee-cell ucee-cell--active"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
      <i
        className="ucee-icon ucee-icon-enter ucee-cell--icon-right"
        style={
          Object {
            "fontSize": "19px",
          }
        }
      />
    </div>
    <div
      className="ucee-cell ucee-cell--active"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
        <div
          className="ucee-cell__label"
        >
          描述信息
        </div>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
      <i
        className="ucee-icon ucee-icon-enter ucee-cell--icon-right"
        style={
          Object {
            "fontSize": "19px",
          }
        }
      />
    </div>
  </div>,
  <div>
    自定义
  </div>,
  <div
    className="ucee-cell-group"
  >
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <div>
          单元格
          <span
            className="ucee-tag ucee-tag--primary"
            style={Object {}}
          >
            标签
          </span>
        </div>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
    <div
      className="ucee-cell ucee-cell--active"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <i
          className="ucee-icon ucee-icon-people"
          style={
            Object {
              "fontSize": "18px",
            }
          }
        />
        <span>
          单元格
        </span>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
      <i
        className="ucee-icon ucee-icon-enter ucee-cell--icon-right"
        style={
          Object {
            "fontSize": "19px",
          }
        }
      />
    </div>
    <div
      className="ucee-cell ucee-cell--active"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <img
          alt=""
          className="ucee-cell__img"
          src="https://img.yzcdn.cn/vant/user-active.png"
        />
        <span>
          单元格
        </span>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
      <i
        className="ucee-icon ucee-icon-enter ucee-cell--icon-right"
        style={
          Object {
            "fontSize": "19px",
          }
        }
      />
    </div>
  </div>,
  <div>
    link
  </div>,
  <div
    className="ucee-cell-group"
  >
    <a
      className="ucee-cell ucee-cell--active"
      href="https://www.baidu.com/"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
    </a>
  </div>,
  <div>
    click
  </div>,
  <div
    className="ucee-cell-group"
  >
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
    </div>
  </div>,
]
`;
